docs/examples/data-shifts.py

```diff
@@ -0,0 +1,83 @@
+"""
+Data Shift Detection & Removal
+===================
+This example covers identifying data shift periods in a time series
+and filtering them out, using the PVAnalytics quality.data_shifts
```

Comment:
It's always nice to use sphinx's cross-referencing syntax (``` :py:mod:`pvanalytics.quality.data_shifts` ```) when referring to a specific part of the package.  In this case it won't actually link anywhere because we don't have a specific definition of the `data_shifts` module, but it'll still format it as code instead of plaintext.